llarp/config/config.cpp

```diff
@@ -107,6 +107,8 @@ namespace llarp
             "private keys.",
         },
         [this](fs::path arg) {
+          if (arg.empty())
```

Comment:
imo this should be a fatal error.  The case where the `data-dir` setting is present in the config file but is an empty string is almost certainly user error and should be treated as such.

Reply:
good point